util/omrutil/j9memclr.cpp

```diff
@@ -57,7 +57,7 @@ OMRZeroMemory(void *ptr, uintptr_t length)
 #endif /* defined(J9ZOS390) || (defined(LINUX) && defined(S390)) */
 
 #if defined(AIXPPC) || defined(LINUXPPC)
-	char *addr = ptr;
+	char *addr = (char *)ptr;
```

Comment:
```suggestion
	char *addr = reinterpret_cast<char*>(ptr);
```

Reply:
I think `static_cast` may be more proper here. I'm not 100% sure it will compile though. Best to check.

Reply:
Should this be `static_cast`?

Reply:
Yes, casting to/from `void*` can be done with `static_cast`.

Reply:
Further down in the file `addr` is being casted as `uintptr_t`, it's happening in quite a few places. Would it better to just change `addr` and `limit` to `uintptr_t *`?

Reply:
No, because it seems to do pointer arithmetic, for example `addr += localCacheLineSize`, and if you change the type of `addr` then you will have to change all such occurrences to the do right thing.